v03_pipeline/lib/misc/vets_test.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+
+import hail as hl
+
+from v03_pipeline.lib.misc.io import split_multi_hts
+from v03_pipeline.lib.misc.vets import (
+    annotate_vets,
+)
+
+
+class VetsTest(unittest.TestCase):
+    def test_annotate_vets(self) -> None:
+        gatk_mt = hl.MatrixTable.from_parts(
+            rows={
+                'locus': [
+                    hl.Locus(
+                        contig='chr1',
+                        position=1,
+                        reference_genome='GRCh38',
+                    ),
+                ],
+                'filters': [
+                    hl.set(['PASS']),
+                ],
+            },
+            cols={'s': ['sample_1']},
+            entries={'HL': [[0.0]]},
+        ).key_rows_by('locus')
+        gatk_mt = annotate_vets(gatk_mt)
+        dragen_mt = hl.MatrixTable.from_parts(
+            rows={
+                'locus': [
+                    hl.Locus(
+                        contig='chr1',
+                        position=1,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=2,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=3,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=4,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=5,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=5,
+                        reference_genome='GRCh38',
+                    ),
+                ],
+                'alleles': [
+                    ['A', 'T'],
+                    ['A', 'T'],
+                    ['A', 'T'],
+                    ['AC', 'T'],
+                    ['AT', 'TC'],
+                    ['AG', 'TG'],
```

Comment:
```suggestion
                    ['AC', 'T'],
                    ['AT', 'ATC'],
                    ['AG', 'ATG'],
```

petty note - don't think variants in Hail can look like that after min-rep, but I'm aware this is super minor to note